slsDetectorServers/eigerDetectorServer/slsDetectorFunctionList.c

```diff
@@ -2155,8 +2137,9 @@ void *start_timer(void *arg) {
 
             // loop packet
             {
+
                 int i = 0;
-                for (i = 0; i != numPacketsPerFrame; ++i) {
+                i != numPacketsPerFrame; ++i) {
```

Comment:
Missing for loop? virtual server doesn't compile now

Reply:
fixed. thanks!